repo/init.go

```diff
@@ -25,7 +25,7 @@ var ErrRepoDoesNotExist = errors.New("repo does not exist, initialization is req
 var ErrMigrationRequired = errors.New("repo needs migration")
 var ErrRepoCorrupted = errors.New("repo is corrupted")
 
-const repover = "6"
```

Comment:
eh, forgot to bump this last time. ndb cause the migration would just be applied on top